services/graph/pkg/config/config.go

```diff
@@ -62,6 +62,8 @@ type LDAP struct {
 	UserNameAttribute        string `yaml:"user_name_attribute" env:"LDAP_USER_SCHEMA_USERNAME;GRAPH_LDAP_USER_NAME_ATTRIBUTE" desc:"LDAP Attribute to use for username of users."`
 	UserIDAttribute          string `yaml:"user_id_attribute" env:"LDAP_USER_SCHEMA_ID;GRAPH_LDAP_USER_UID_ATTRIBUTE" desc:"LDAP Attribute to use as the unique ID for users. This should be a stable globally unique ID like a UUID."`
 	UserEnabledAttribute     string `yaml:"user_enabled_attribute" env:"LDAP_USER_ENABLED_ATTRIBUTE;GRAPH_USER_ENABLED_ATTRIBUTE" desc:"LDAP Attribute to use as a flag telling if the user is enabled or disabled."`
+	LocalUserDisable         bool   `yaml:"local_user_disable" env:"LDAP_LOCAL_USER_DISABLE;GRAPH_LOCAL_USER_DISABLE" desc:"Boolean to control behavior for disabling users. If true, disabling a user via API will add the user to a group for disabled users."`
```

Comment:
I am sorry, but while playing with this, I just realized that we need to make this setting a bit more flexible.
Basically we need to have at least 3 ways to configure enabled/disabled users.

I'd suggest to have one setting to configure the mechanism how the enabled/disabled state of a user is determinded.

`DISABLE_USER_MECHANISM`: For now it should support 3 different values: `none`, `attribute`, `group`

Where:
`none`: users won't have the `accountEnabled` attribute. This is needed for being able to integrate with external read-only LDAP servers. (Where we don't have any possiblity to maintain a enabled attribute or some local groupmembership)

`attribute`: use the `UserEnabledAttribute` to set the `accountEnabled` property

`group`: use the` LocalUserDisableGroupDN` to set the `accountEnabled` property.

Making this a string valued options also has the advantage of being able to add more enable/disable mechanism later on. E.g. for evaluating the ActiveDirectory `userAccountControl` attribute bitmask.